internal/api/http.go

```diff
@@ -220,6 +222,32 @@ func Start(cfg *config.Config, app *core.Core, docker *client.Client, z *zfs.ZFS
 		return c.JSON(http.StatusOK, res)
 	})
 
+	e.POST("/resources/:resource/claim", func(c echo.Context) error {
+		resource := c.Param("resource")
+		timeoutStr := c.QueryParam("ttl")
+		timeout := internal.DefaultClaimTimeoutSeconds * time.Second
+		if timeoutStr != "" {
+			t, err := strconv.ParseInt(timeoutStr, 10, 64)
+			if err == nil {
+				timeout = time.Duration(t) * time.Second
+			}
+		}
+
+		clone, err := app.ClaimPooledClone(resource, timeout)
```

Comment:
Shouldn't we add `c.Get("owner")` as a parameter to this also, and also for delete and check when a "user" tries to expire a clone, that it's actually the owner of that clone?

Reply:
Probably yes. I'm wondering what should happen when it's used anonymously, for example in an integration test where I haven't set up any zdap config.

Reply:
I think that `zdap` should require to have a user to be able to clone/claim anything, so that we can see the owners on the zdap servers. We need the zdap server IP's so to add a user to that shouldn't be a big problem as I see it..

Reply:
As discussed, let's try to let the zdap cli accept a config file path